polkadot/node/core/provisioner/src/lib.rs

```diff
@@ -77,13 +77,12 @@ impl ProvisionerSubsystem {
 
 /// Per-session info we need for the provisioner subsystem.
 pub struct PerSession {
-	elastic_scaling_mvp: bool,
+	prospective_parachains_mode: ProspectiveParachainsMode,
```

Comment:
why are you adding this back?
Looks like there is no longer any per session data that needs to be stored, so we can remove this altogether